charts/vela-core/templates/defwithtemplate/webservice.yaml

```diff
@@ -136,7 +136,7 @@ spec:
         	// +short=p
         	port: *80 | int
         
-        	// If addRevisionLabel is true, the appRevision label will be added to the underlying pods
+        	// +ignore If addRevisionLabel is true, the appRevision label will be added to the underlying pods
```

Comment:
This seems a little bit weird as we normally put description in annotation `//+usage=xxx`.